### PR TITLE
airflow for new comers

### PR DESCRIPTION
**Pull request by ephraimbuddy**
My blogpost draft

### CODE REVIEW

File: landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+title: "Apache Airflow For NewComers"
```
Comment (ephraimbuddy) [inline landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md:2]:
```suggestion
title: "Apache Airflow For Newcomers"
```

File: landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+linkTitle: "Apache Airflow For NewComers"
```
Comment (ephraimbuddy) [inline landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md:3]:
```suggestion
linkTitle: "Apache Airflow For Newcomers"
```

**Comment by ephraimbuddy** [comment]
@turbaszek can I ask for a review?

File: landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+[about me](https://ephraimbuddy.wordpress.com/2020/05/24/introduction/)
+ and [how I got into](https://ephraimbuddy.wordpress.com/2020/05/06/experiences-applying-to-outreachy/) the Outreachy Program.
+
+It has been an amazing experience! Thanks to my mentors [Jarek](https://github.com/potiuk) and
+[Kaxil](https://github.com/kaxil), and the community members especially [Kamil](https://github.com/mik-laj)
+and [Tomek](https://github.com/turbaszek) for all their supports. I'm grateful!
+
+
+
+
```
Comment (ephraimbuddy) [inline landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md:105]:
```suggestion
```

**Comment by potiuk** [comment]
cc: @leahcole as well :)

File: landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+to the excellent [documentation on how to get started](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst).
+
+I joined the community about 8 weeks ago through the [Outreachy Program](https://www.outreachy.org/) and have
+completed about [30 PRs](https://github.com/apache/airflow/pulls/ephraimbuddy). How fast time flies! You can read
+[about me](https://ephraimbuddy.wordpress.com/2020/05/24/introduction/)
+ and [how I got into](https://ephraimbuddy.wordpress.com/2020/05/06/experiences-applying-to-outreachy/) the Outreachy Program.
+
+It has been an amazing experience! Thanks to my mentors [Jarek](https://github.com/potiuk) and
+[Kaxil](https://github.com/kaxil), and the community members especially [Kamil](https://github.com/mik-laj)
+and [Tomek](https://github.com/turbaszek) for all their supports. I'm grateful!
```
Comment (leahecole) [inline landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md:101]: nit - support, not supports :)

**Comment by ephraimbuddy** [comment]
Hi @leahecole , thanks so much for the reviews. Can you also take a look at #274 , I'm leaning towards it. I'm preferring it to be the one that would be published. I see that most of the things I wrote here are already available on the Documentation. Sorry for the stress in reviewing this. Thanks once more

**Comment by leahecole** [comment]
> Hi @leahecole , thanks so much for the reviews. Can you also take a look at #274 , I'm leaning towards it. I'm preferring it to be the one that would be published. I see that most of the things I wrote here are already available on the Documentation. Sorry for the stress in reviewing this. Thanks once more

You have nothing to apologize for! No stress here :) and I think that it's totally fine to link to documentation in blog posts - imho, blog posts tell a story and are sometimes more high level, and by linking to docs, the reader can choose to learn more by clicking the link, but they don't have to if they don't want to. I'll go look at the other one now!

**Comment by ephraimbuddy** [comment]
> You have nothing to apologize for! No stress here :) and I think that it's totally fine to link to documentation in blog posts - imho, blog posts tell a story and are sometimes more high level, and by linking to docs, the reader can choose to learn more by clicking the link, but they don't have to if they don't want to. I'll go look at the other one now!
>
Alright, I will apply the suggestions. Thanks!

**Comment by leahecole** [comment]
Review on #274 done!

File: landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+monitor and troubleshoot DAGs and Tasks.
+
+There are many actions you can perform on the UI. You can trigger a task, monitor the execution
+including the duration of the task. The UI makes it possible to view the task's dependencies in a
+tree view and graph view. You can view task logs in the UI.
+
+The web UI is started with the command `airflow webserver` in the breeze environment.
+
+### Backend
+By default, Airflow uses the SQLite backend for storing the configuration information, DAG states,
```
[the remark this answers is not in the document]
Comment (ephraimbuddy) [reply landing-pages/site/content/en/blog/apache-airflow-for-new-comers.md:67]: I think it is both the breeze default and the default every time breeze is used for the first time unless configured to use Postgres or MySQL.

**Comment by ephraimbuddy** [comment]
Hi @leahecole, Please take a look again, I might have omitted some suggestions. I moved the blog post to a folder so that the images will be in one place. Thanks.

**Comment by potiuk** [comment]
Hey @leahecole -> as the internship is coming to and end, it would be great you re-review it :)

**Comment by potiuk** [comment]
🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 

@kaxil @turbaszek @ephraimbuddy -> once it builds, we can promote it

**Comment by kaxil** [comment]
Wohoo  🚀 🚀🚀

**Comment by potiuk** [comment]
Somehow the images are not showing up - so let's wait with promotion 😕 

@mik-laj -> how do we add the images to be displayed? Should we move them to static ? And how to refer to them. I could not find an example /doc explaining it.

**Comment by mik-laj** [comment]
@potiuk  We use Hugo for content management.   It might be helpful for us.
https://discourse.gohugo.io/t/solved-how-to-insert-image-in-my-post/1473

**Comment by kaxil** [comment]
I used folder to manage all the dependent files:


Example:
https://github.com/apache/airflow-site/tree/master/landing-pages/site/content/en/blog/airflow-1.10.10
image

**Comment by potiuk** [comment]
Thank @Kaxil. works like a charm !

Screenshot from 2020-08-17 22-04-18

Approval needed :) #281

**Comment by ephraimbuddy** [comment]
🚀🚀🚀🚀🚀🚀

**Pull request merged**